content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Use Arduino with Chromebook"
+---
+
+You can write and upload sketches on Chrome OS with the Web Editor in Arduino Cloud. The Web Editor has an extensive feature set similar to the desktop IDE. While many popular boards are compatible, you should check [the list of compatible boards here](https://support.arduino.cc/hc/en-us/articles/360014779899).
+
+---
+
+## Install the Chrome app
+
+1. Go to the [Chrome web store page](https://chrome.google.com/webstore/detail/elmgohdonjdampbcgefphnlchgocpaij).
+2. Click _Add to Chrome_.
+3. Click _Add app_ to confirm.
+4. Click _Launch app_ to start the Web Editor or go to [create.arduino.cc/editor](https://create.arduino.cc/editor).
+
+## Use the Web Editor
+
+1. Go to [create.arduino.cc/editor](https://create.arduino.cc/editor).
+2. If you're not signed into an Arduino account, you will be asked to do so.
+3. Connect a [compatible board](#compatible-boards) to a USB port on your computer.
```

Comment:
Maybe this can link to the compatible boards page instead?
```suggestion
3. Connect a [compatible board](https://support.arduino.cc/hc/en-us/articles/360014779899) to a USB port on your computer.
```

Reply:
That was a mistake, you're right of course 👍